tools/scripts/getWorkerInfo.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+#
+# Copyright (c) 2022, NVIDIA CORPORATION. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# This script copies and executes discoveryScript.sh to the worker and copies
+# back the generated YAML file. The YAML file is used by the AutoTuner for
```

Comment:
add assumes ./discoveryScript.sh  from where the script is run

Reply:
Added assumption.